src/pages/wishListView/wishListComponent/wishlistComponent.js

```diff
@@ -80,7 +104,7 @@ const WishlistCardComponent = ({ props, onClick }) => {
               </Typography>
             }
             action={
-              <IconButton className={classes.icon} onClick={onClick}>
+              <IconButton className={classes.icon} onClick={handleDelete}>
                 <DeleteOutline />
```

Comment:
Hey, @RohankumarM  actually there is a bit issue here you have made `handleDelete ` function which replaces the onClick prop because of this we are unable to use the  dispatch `wishlistDeleted ` action which is coming from  `src\pages\wishListView\index.js` through the onClick prop.
```javascript

 <WishlistCardComponent
                key={item.id}
                props={item}
                onClick={() => wishlistDeleted({ id: item.id })}
              />
```
Please resolve that @RohankumarM  :)

Reply:
Yes, I forgot to add callback to onClick props. I have added into handleDelete function.

Reply:
Yes I have called the props onClick in the handleDelete function, now it dispatches and updates the state. Thanks for reminding me. :)